_pages/guidelines/web/write-effective-links.md

```diff
@@ -20,7 +20,7 @@ sources:
     publisher: W3C
   - title: '18F Content Guide: Link Text'
     link: 'https://content-guide.18f.gov/urls-and-filenames/#link-text'
-  - title: U.S. Web Design System: Links
+  - title: 'U.S. Web Design System: Links'
   - link: https://designsystem.digital.gov/components/link/
```

Comment:
The other link values also use quotes for the `link`. Could you add and see if that fixes the formatting issue that's rendered in **Sources** in [Federalist preview](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.app.cloud.gov/preview/gsa/plainlanguage.gov/nl-yaml-syntax-error/guidelines/web/write-effective-links/)?

![image](https://user-images.githubusercontent.com/3385219/168657391-db241ad5-8481-4366-9e89-4df16c816e2b.png)